### PR TITLE
Update github extension

### PR DESCRIPTION
**Pull request by xilopaint**
## Description

- Updated IDE icons.

## Screenshot

Screenshot 2024-08-07 at 13 46 14

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/github/assets/vscode.svg
Comment (LitoMore) [inline extensions/github/assets/vscode.svg:1]: The icon size is not consistent with others. You can use this one instead:



https://cdn.jsdelivr.net/npm/simple-icons@12/icons/visualstudiocode.svg

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: extensions/github/assets/vscode.svg
Comment (xilopaint) [reply extensions/github/assets/vscode.svg:1]: Done.

**Review by LitoMore — APPROVED** [review]

File: extensions/github/assets/sourcegraph.svg
Comment (LitoMore) [inline extensions/github/assets/sourcegraph.svg:1]: Ah, also this one. We use the monochrome version of the icon:



https://cdn.jsdelivr.net/npm/simple-icons@9/icons/sourcegraph.svg

This is the version that was previously on Simple Icons. It has the same image size and quality as the other icons.

Comment (xilopaint) [reply extensions/github/assets/sourcegraph.svg:1]: Done.

**Review by jfkisafk — APPROVED** [review]

**Review by jfkisafk — APPROVED** [review]

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**